src/internal/operations/restore.go

```diff
@@ -316,5 +320,5 @@ func formatDetailsForRestoration(
 		paths[i] = p
 	}
 
-	return paths, nil
+	return paths, errs.Err()
```

Comment:
same here, this can hide errors and the current code may not be setup to deal with that